app/src/main/res/values-it/strings.xml

```diff
@@ -233,13 +233,13 @@ ed anonima"</string>
     <string name="bluetooth_permission_turned_off">"La App non funziona perché non è presente l'autorizzazione Bluetooth."</string>
 
     <!-- Permission Text für Bluetooth -->
-    <string name="NSBluetoothPeripheralUsageDescription">"Gli incontri vengono acquisiti in background tramite Bluetooth."</string>
+    <string name="NSBluetoothPeripheralUsageDescription">"Le co-presenze vengono conteggiate in background tramite il Bluetooth."</string>
```

Comment:
Le co-presenze vengono rilevate in background tramite il Bluetooth

Reply:
corretto.